uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/PieChartPopupMediaItem.h

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ *  Copyright 2012-2025 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#ifndef ESRI_ARCGISRUNTIME_TOOLKIT_PIECHARTPOPUPMEDIAITEM_H
+#define ESRI_ARCGISRUNTIME_TOOLKIT_PIECHARTPOPUPMEDIAITEM_H
+
+// Qt headers
+#include <QJsonArray>
+#include <QObject>
+
+// Other headers
+#include "PopupMediaItem.h"
+
+class QPieSlice;
+
+namespace Esri::ArcGISRuntime {
+
+namespace Toolkit {
```

Comment:
Unless there are Maps SDK forward declares to insert, just combine the namespaces

```c++
namespace Esri::ArcGISRuntime::Toolkit {
```

More changes needed at the bottom of the file.